src/reloader.cpp

```diff
@@ -134,11 +134,14 @@ void Reloader::reload(unsigned int pos,
 					_("Error while retrieving %s: %s"),
 					utils::censor_url(oldfeed->rssurl()),
 					emsg);
-		} catch (rsspp::Exception& e) {
+		} catch (const rsspp::Exception& e) {
 			errmsg = strprintf::fmt(
 					_("Error while retrieving %s: %s"),
 					utils::censor_url(oldfeed->rssurl()),
 					e.what());
+		} catch (const rsspp::NotModifiedException&) {
+			// Nothing to be done, feed was not chaned since last retrieve
+			oldfeed->set_status(DlStatus::SUCCESS);
```

Comment:
Needs to be set because otherwise status will be kept as `DURING_DOWNLOAD`.
Can be seen with a config option like: `feedlist-format "%S %4i %n %11u %t"`